src/legacy/patchHeaders.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Patch 'Headers' to accept 'undefined'.
+ * Fixes `TypeError: Failed to construct 'Headers': No matching constructor signature.`
+ * Affected platforms:
+ *   - Tizen 3
+ *   - Tizen 4
+ *   - webOS 4
+ */
+
+(function (window) {
+    'use strict';
+
+    if (window.Headers) {
+        try {
+            new window.Headers(undefined);
+        } catch (_) {
+            console.debug('patch \'Headers\' to accept \'undefined\'');
+
+            const _Headers = window.Headers;
+
+            window.Headers = function (init) {
+                return init ? new _Headers(init) : new _Headers();
```

Comment:
I thought this needed to pass an empty object if no arguments are passed to the constructor?

Reply:
It is optional: https://developer.mozilla.org/en-US/docs/Web/API/Headers/Headers
And it seems to work.
That [react-router fix](https://github.com/remix-run/react-router/pull/11148) also uses an empty constructor.